src/data/seq/seq.lean

```diff
@@ -62,6 +73,9 @@ theorem le_stable (s : seq α) {m n} (h : m ≤ n) :
   s.1 m = none → s.1 n = none :=
 by {cases s with f al, induction h with n h IH, exacts [id, λ h2, al (IH h2)]}
 
+/-- If a sequence terminates at position `n`, it also terminates at `m ≥ n `. -/
+def terminates_stable := @le_stable
```

Comment:
Give the explicit type?

Should this be a lemma, rather than a def?

Reply:
Ask `#sanity_check`.